manifests/kustomize/env/aws/params.env

```diff
@@ -0,0 +1,6 @@
+dbHost=YOUR_RDS_ENDPOINT
+mlmdDb=rds
+
+bucketName=YOUR_S3_BUCKET_NAME
+minioServiceHost=s3.amazonaws.com
+minioServiceRegion=us-west-2
```

Comment:
You need to tell user update S3 region as well. It can not be fixed `us-west-2`

Reply:
Make sense, will fix.